modules/data_release/test/TestPlan.md

```diff
@@ -1,10 +1,12 @@
 # Data Release Module Test Plan 
```

Comment:
valuable additions to this file:

 - granting permission to aversion from the Add permission button is not covered
 - try to add permission to both a file and a version, an error should be thrown.
 - in the permissions section, item 10 should come right before 8. otherwise it gets confusing that an unchecked box does not actually revoke permissions
- add test for download CSV
- another fun usecase is to test multiple version "names". We had a bug a couple of versions ago where version names with a `.` in them were breaking the module (more specifically managing permissions)